docs/source/schemas/index.rst

```diff
@@ -133,33 +133,35 @@ characteristics, among others. The types are:
 +-------------------------------+--------------+---------------+----------------------+----------------+
 | NULLABLE\_FIXED\_LONG         | long         | byte[9]       | YES                  | YES            |
 +-------------------------------+--------------+---------------+----------------------+----------------+
-| VAR\_LONG                     | long         | byte[len]\*   | YES                  | YES            |
+| VAR\_LONG                     | long         | byte[len] \*  | YES                  | YES            |
++-------------------------------+--------------+---------------+----------------------+----------------+
+| VAR\_SIGNED\_LONG             | long         | byte[len] \*  | YES                  | YES            |
 +-------------------------------+--------------+---------------+----------------------+----------------+
-| VAR\_SIGNED\_LONG             | long         | byte[len\*]   | YES                  | YES            |
+| UUID                          | UUID         | byte[16]      | YES                  | YES            |
```

Comment:
However, I can see that it's not obvious from the docs why we have both FIXED_LONG and FIXED_LONG_LITTLE_ENDIAN.  I found the following comment in the code:

"This type is good if you don't need to support range scans because keys next to each other won't be written next to each other.  This is good because it will spread out the load of writes to many different ranges."

I'm not convinced if this is interesting/relevant enough to mention, but let me know if you think that kind of explanation would be useful here and I'm happy to add it as a note.